File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Set<Integer> num_set = Arrays.stream(nums)
                            .boxed()
                            .collect(Collectors.toSet());
        int res = 0;
        for(int num: num_set){
            if(num_set.contains(num - 1) == false){
                int len = 0;
                while(num_set.contains(num+len)){
                    len+=1;
                }
                res = Math.max(len, res);
            }
        }
        return res;
    }
}

File: Data Structures & Algorithms/maximum-subarray/submission-0.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:
        totalSum = nums[0]
        localSum = 0
        for num in nums:
            localSum += num 
            totalSum = max(totalSum, localSum)
            if localSum < 0:
                localSum = 0
            
        return totalSum 